bittensor/_dataset/__init__.py

```diff
@@ -36,8 +36,10 @@ def __new__(
             max_corpus_size:int = None,
             num_workers: int = None,
             dataset_name: list = [],
-            save_dataset: bool=None,
-            no_tokenizer: bool=None
+            save_dataset: bool = None,
+            no_tokenizer: bool = None,
+            world_size: int = 0,
```

Comment:
What is world_size used for? I don't see it being used anywhere

Reply:
Hahaha omg nice catch thank you!